src/patternfly/components/Menu/menu.scss

```
@@ -567,6 +568,10 @@
     padding-top: var(--pf-c-menu__list-item--m-loading--PaddingTop);
     overflow: hidden; // prevents spinner rotation from overflowing
   }
+
+  &.pf-m-danger {
+    --pf-c-menu__list-item--Color: var(--pf-c-menu__list-item--m-danger__item--Color, inherit);
```

```suggestion
    --pf-c-menu__item--Color: var(--pf-c-menu__list-item--m-danger__item--Color, inherit);
```

small change - you want to make the item color red - similar to load. https://github.com/patternfly/patternfly/blob/736e343ceecb31893c00afeef3593c486e273437/src/patternfly/components/Menu/menu.scss#L557-L560

---

Corrected